STHAL - Carthage Support

ConEliopoulos opened the pull request:
Updated for Carthage support

@TabDigital/mobile

cysp commented on Cartfile.resolved:
```diff
@@ -0,0 +1 @@
+github "TabDigital/STURITemplate" "863d5b8390d3c3ae87eab6301f136b61c179dcf3"
```
Can we `.gitignore` this file so that travis always builds against the default branch of `STURITemplate`, and we don't have to update everything through the web of dependencies when a core derp gets updated?

cysp commented:
`.travis.yml` needs an update to get builds on Travis working.

ConEliopoulos commented:
`Git`, what are you doing!

For some reason, git has included `Eggify` as part of this pull request. Not sure why/how this has happened. On my local branch which is `up-to-date with 'origin/carthage'`, running

`git log master..carthage --oneline`

gives

`a734713 Configure for Travis builds.
9932658 Create README.md
a47fec3 Fix tests.
90d6829 Updated visibility of headers.
a09c999 Update umbrella header.
af0e4d3 Share iOS framework scheme.
ff0e3af Support Carthage`

No `Eggify` there!?

cysp commented:
I think I might have done that. I did some things to master (because master on this repo wasn't actually what we were targeting from the app) and I guess I confused GitHub.

ConEliopoulos commented:
I just re-cloned the repo, deleted the commit and rebased. Everything seemed to go ok and the git history looks good now. But, Xcode has now 💩 itself. I'm getting `unused parameter` errors with `enumerateKeysAndObjectsUsingBlock`. Config looks good. It's saying to ignore this and `build settings -> levels` always leads to `NO`. I bet if I try again tomorrow it will build just fine...
